addons/event_sale/tests/test_event_sale.py

```diff
@@ -473,3 +473,35 @@ def test_cancel_so(self):
         self.sale_order._action_cancel()
         self.assertEqual(len(event.registration_ids), 1)
         self.assertEqual(event.registration_ids.state, 'cancel')
+
+    @users('user_sales_salesman')
+    def test_adjusted_quantity_in_sale_order(self):
```

Comment:
Please try to keep the tests somewhat ordered alphabetically. Even if the one above seems to have been forgotten too.